jobs/jobs.go

```diff
@@ -90,6 +91,15 @@ func (repository *CareerRepository) GetCareers() (careersList []Career, err erro
 											   apply_seo_title, apply_seo_description
 											   FROM jobs WHERE is_active = 1 `)
 
+	var uniqueId string
```

Comment:
We should create new API for this script, otherwise it will execute in all API calls

Reply:
No need to create separate API for that because if condition in true that time only call update query

Reply:
We have to create separate API for this type of change, in future if we want to disable that API , we can do that

Reply:
okay